components/Profile/Page.js

```diff
@@ -0,0 +1,441 @@
+import React, { Component, Fragment } from 'react'
```

Comment:
Just wondering if a `git mv` from index.js would have saved us some review diff here.

Reply:
not sure but would probably have marked almost all lines as changed anyhow